l10n_ch_adr_report/report/dangerous_delivery_report.py

```diff
@@ -116,19 +113,10 @@ def _get_DG_move_line_vals(self, moves):
         # unit measurement on stock is not considered
         result = []
         for product, move_lines in moves:
-            qty = 0
-            for rec in move_lines:
-                if rec.state == "done":
-                    qty += rec.quantity_done
-                else:
-                    qty += rec.product_uom_qty
+            qty = sum(line.quantity for line in move_lines)
```

Comment:
I am not sure this is correct, because with the change in Odoo core we still have the `product_uom_qty` field.
I think this
```
qty = 0
for rec in move_lines:
    if rec.picked:
        qty += rec.quantity
    else:
        qty += rec.product_uom_qty
```

Reply:
thanks @TDu, i see that the use of picked field is not quite like [its definition](https://github.com/odoo/odoo/blob/5bb1f22984506188f5300ae765a70b813f32dcad/addons/stock/models/stock_move.py#L125) in this case. but maybe i was a bit confused about product_uom_qty so i will restore the original code and just edit quantity to replace the outdated quantity_done